src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -4,11 +4,11 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-    <VersionPrefix>12.0.0-preview</VersionPrefix>
+    <VersionPrefix>12.0.1-preview</VersionPrefix>
```

Comment:
@ritwikbasu remove this and replace here with < version > but don't keep both tags.
Also move the < version > tag to the top property group and not the propertygroup below.

Also you need to bump up the assembly version in AssemblyInfo.cs